main/src/com/google/refine/importers/TabularImportingParserBase.java

```diff
@@ -161,6 +163,9 @@ static public void readTable(
                             
                             Object value = cells.get(c);
                             if (value instanceof Cell) {
+                                if(trimStrings) {
+                                    value = ((Cell) value).value.toString().trim();
```

Comment:
If `value` is an instance of `Cell`, then you should not replace it by a `String`. This will cause runtime type errors.